src/script/components/MessagesList/Message/ContentMessage/Warnings/PartialFailureToSend/PartialFailureToSend.tsx

```diff
@@ -60,23 +61,41 @@ function generateNamedUsers(users: User[], userClients: QualifiedUserClients): P
   );
 }
 
+function generateUnreachableUsers(users: QualifiedId[]) {
+  const userCountByDomain = countBy(users, 'domain');
+  return map(userCountByDomain, (count, domain) => ({count, domain}));
+}
+
+function reduceWithCommas(elements: React.ReactNode[]) {
```

Comment:
The naming is a little `off` reducing is not what the consumer wants to do. 
What they want to do is to `joinWith` and give it a custom separator :) 

```suggestion
function joinWith(elements: React.ReactNode[], separator: string) {
```

WDYT?

Reply:
wasn't happy with the naming and the custom separator sounds good :+1: